core/store/src/columns.rs

```diff
@@ -139,6 +139,7 @@ pub enum DBCol {
     /// Mapping from Receipt id to destination Shard Id, i.e, the shard that this receipt is sent to.
     /// - *Rows*: ReceiptId (CryptoHash)
     /// - *Content type*: Shard Id || ref_count (u64 || u64)
+    /// DEPRECATED: DO NOT use this column. This column is not written or read in production.
     ReceiptIdToShardId,
```

Comment:
nit: The other deprecated columns are renamed to include "_" suffix and with the `strum(serialize = OldName)`. Can you follow the same convention?  Also I would consider removing the description or moving the deprecated to be the first thing in the description.

Reply:
done.